src/utils/SearchUtils.js

```diff
@@ -1,12 +1,21 @@
 // @flow
+import fuzz from 'fuzzball';
 import LRU from 'lru-cache';
+import AllStopUtils from './AllStopUtils';
 import RequestUtils from './RequestUtils';
 import Constants from './Constants';
 
+const BUS_STOP = 'busStop';
+const MIN_FUZZ_RATIO = 75;
 const placeIDToCoordsCacheOptions = {
   max: 10000, // Maximum size of cache
 };
 const placeIDToCoordsCache = LRU(placeIDToCoordsCacheOptions);
+const queryToPredictionsCacheOptions = {
```

Comment:
why don't you just combine the two caches into one

Reply:
the current cache in `/search` stores Google specific info like `placeID` and we have to keep those for users who don't update to this new version when we push it out.

Reply:
i understand, but what i meant is couldn't you just have one cache that holds lots of info per query including placeid?

Reply:
oh i understand what you mean. lets say we have a query `Duff` and and the user who searches it is using Apple MapKit. Then, inside the cache, we would have `Duff: [{name, lat, long}...]` w/o `placeID`'s. Lets say the next user then queries `Duff` but has not updated and is using `GoogleMaps`, if we were using the same cache we would have to check if the values for `Duff` include `placeID`. If they don't, as in this case, then we would have to do the GoogleMaps lookup and overwrite the cache. My thinking was just that since the ApplePlaces and GooglePlaces objects contain different keys, having it as two separate caches made sense. lmk if that sounds reasonable or not